### DISCUSSION TITLE
Is the leaderboard results all Pass@1 for HumanEval?

### DISCUSSION THREAD

**Discussion by zhimin-z** [open-compass/opencompass#687] [Q&A] [answered]
https://opencompass.org.cn/dataset-detail/HumanEval

**Comment by yingfhu** [comment, marked as answer]
Yes, the leaderboard only gives pass@1.  We also support pass@k eval following https://opencompass.readthedocs.io/en/latest/advanced_guides/code_eval.html

**Reply by zhimin-z** [reply]
Thanks, hopefully this configuration could be added to the existing documentation :)
